setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "K3s"
+description: Set up K3s on an Ubuntu machine to deploy Coder.
+---
+
+This article will show you how to install K3s onto a new Ubuntu 20.04 LTS
+machine for use with Coder.
+
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that works well
+for single-node or multi-node clusters. This guide covers the installation of
+K3s onto a new Ubuntu 20.04 LTS machine. If you want to install Coder on a local
+machine or an existing host, a [kind cluster](./kind.md) or
+[k3d cluster](https://k3d.io/) may be a better choice, as it leverages Docker to
+set up/tear down clusters with little hassle.
+
+> This installation method is not officially supported or tested by Coder. If
+> you have questions or run into issues, feel free to reach out using our
+> [community Slack channel](https://cdr.co/join-community).
+>
+> **We do not recommend using K3s for production deployments of Coder.**
+
+## Prerequisites
+
+Before proceeding, please make sure that:
+
+1. You have an **Ubuntu 20.04 machine**: This can be a bare metal or a virtual
+   machine.
+
+   Ensure that the machine's specs satisfy Coder's
+   [resource requirements](../requirements.md), since your experience with Coder
+   is dependent on your system specs.
+
+1. You have the following software installed on your machine:
+
+   - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+   - [helm](https://helm.sh/docs/intro/install/)
+
+1. Your network policy or firewall accepts incoming traffic on:
+   - Port 22 (SSH)
+   - Port 80 (HTTP)
+   - Port 443 (HTTPS)
+   - Port 5349 (TURNS)
+   - **Optional**: Port 8443 (Kubernetes API)
+
+## Step 1: Change the default SSH port
+
+If you enable Networking v2 after installing Coder (in
+`Manage > Admin > Infrastructure`), this step is not necessary to SSH into
+workspaces, as TURNS is used instead.
```

Comment:
@deansheather added a note :)

I noticed that networking v2 is not mentioned anywhere else in docs so I specifically referenced how this can be done.